plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -182,21 +192,18 @@ def raw_execute(self, sql, fetch=False):
 
         logger.debug('On {}: {}', conn.name, sql)
 
-        job_config = google.cloud.bigquery.QueryJobConfig()
-        job_config.use_legacy_sql = False
+        job_params = {'use_legacy_sql': False}
 
-        priority = conn.credentials.priority
+        priority = conn.credentials.get('priority', 'interactive')
```

Comment:
```suggestion
        priority = conn.credentials.priority
```
The rest of this change where you're converting the job params to a dict is fine, but this should be updated to 0.15.x syntax.

Reply:
Should be done!